sdk/servicebus/Azure.Messaging.ServiceBus/tests/Client/ServiceBusClientTests.cs

```diff
@@ -473,11 +473,6 @@ public void MetricsPropertyDoesNotThrowWhenEnabled()
         ///
         public class ReadableOptionsMock : ServiceBusClient
         {
```

Comment:
We should consider some unit tests for the handlers here, since we can fully control when they fire and what args they emit.